src/main/java/com/leggeart/legge/LawsuitRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.leggeart.legge;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+/**
+ Dao class to handle Lawsuit class.
```

Comment:
DAO is not a repository 
https://www.baeldung.com/java-dao-vs-repository#:~:text=DAO%20works%20as%20a%20data,be%20implemented%20using%20a%20repository.